cachi2/core/package_managers/pip.py

```diff
@@ -2018,12 +2029,12 @@ def _download_url_package(
     download_to = pip_deps_dir.join_within_root(_get_external_requirement_filepath(requirement))
     download_to.path.parent.mkdir(exist_ok=True, parents=True)
 
-    if url.hostname in trusted_hosts:
-        log.debug("Disabling SSL verification, %s is a --trusted-host", url.hostname)
-        insecure = True
-    elif url.port is not None and f"{url.hostname}:{url.port}" in trusted_hosts:
+    if url.port is not None and f"{url.hostname}:{url.port}" in trusted_hosts:
         log.debug("Disabling SSL verification, %s:%s is a --trusted-host", url.hostname, url.port)
         insecure = True
+    elif url.hostname in trusted_hosts:
+        log.debug("Disabling SSL verification, %s is a --trusted-host", url.hostname)
```

Comment:
Is there any reason for not using f-strings in log messages?

Reply:
IIRC, it's because %s strings are lazy-eval; in any case, that decision was made before my time.

Reply:
Got it, thank you!

Reply:
I'm not a fan either and I like consistency, so to just add some minor context to this, I myself was recently pointed out the old string formatting style for logging is preferable and even recommended (though the official docs weren't that clear on this to me), but the truth is that this is not the regular old-style Python string formatting since the substitution happens in the logger, i.e. not immediately, so if you have a lot of debug logs, the potential penalty here is the added formatting burden of very large (e.g. JSON) strings that would otherwise not be used at all in production where debugging is disabled, which to me was really the only acceptable argument for this - we haven't formalized a guideline on this though yet.